back/config/initializers/filter_parameter_logging.rb

```diff
@@ -2,11 +2,11 @@
 
 # Be sure to restart your server when you modify this file.
 
-# Configure parameters to be filtered from the log file. Use this to limit dissemination of
-# sensitive information. See the ActiveSupport::ParameterFilter documentation for supported
-# notations and behaviors.
+# Configure parameters to be partially matched (e.g. passw matches password) and filtered from the log file.
+# Use this to limit dissemination of sensitive information.
+# See the ActiveSupport::ParameterFilter documentation for supported notations and behaviors.
 Rails.application.config.filter_parameters += %i[
-  password current_password passw secret token _key crypt salt certificate otp ssn import_ideas.pdf import_ideas.xlsx
+  passw secret token _key crypt salt certificate otp ssn import_ideas.pdf import_ideas.xlsx
```

Comment:
Deliberate change to remove `password` and `current_password` from this list?

Reply:
Yep, according to the documentation, it's supposed to use partial matching. So, `password` and `current_password` are covered by `passw`.